src/main/scripts/org/reaktivity/specification/amqp/session/simultaneous.end.exchange/client.rpt

```diff
@@ -0,0 +1,170 @@
+#
```

Comment:
Please rename this scenario to `end.exchange.simultaneous`.
Note: if we have a similarly named `close.exchange` scenario then please make sure it is called `close.exchange.simultaneous`.